ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, seed, brain, trainer_parameters):
         self.seed = seed
         self.brain = brain
         self.use_recurrent = trainer_parameters["use_recurrent"]
+        self.normalize = trainer_parameters.get("normalize", False)
```

Comment:
Don't you want this to default to True?

Reply:
Default is False as normalize modifies the observation values

Reply:
I'm actually a little torn on what the default should be longer term since most users will likely need to normalize their vector obs -- but I think that for now it makes sense to stay with the more conservative choice.